Add testing instructions to Readme (#7737)
DataDog/datadog-agent: README.md

Please refer to the [Agent Developer Guide](docs/dev/README.md) for more details. For instructions
on setting up a windows dev environment, refer to [Windows Dev Env](devenv).

## Testing

> Run tests using `invoke test`. During development add the `--skip-linters` option to skip straight to the tests. 

Reviewer: ```suggestion
Run tests using `invoke test`. During development, add the `--skip-linters` option to skip straight to the tests. 
```

```
invoke test --targets=./pkg/aggregator/... --skip-linters
```

> When testing code that depends on [rtloader](/rtloader) make sure to first build & install it.

Reviewer: ```suggestion
When testing code that depends on [rtloader](/rtloader), build and install it first.
```

```

> invoke rtloader.make && invoke rtloader.install

Reviewer 2: Instead of using `invoke rtloader.make && invoke rtloader.install` you can use `invoke agent.build` which call `invoke rtloader.make && invoke rtloader.install`.
